src/utils/index.ts

```diff
@@ -178,3 +178,25 @@ export const provider2SVG = (providerName: ProviderName | ProviderType) => {
       return 'img/icon_metamask3.svg'
   }
 }
+
+export const web3CompatibleNetwork = () => {
+  if (typeof window === 'undefined' || !window.web3) return 'UNKNOWN'
+
+  let netID
+
+  // 1.X.X API
+  if (typeof window.web3.version === 'string') {
```

Comment:
Should this be `typeof window.web3.version !== 'string`?

Reply:
nay